android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -2,4 +2,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.9-all.zip
\ No newline at end of file
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.6.4-all.zip
```

Comment:
This change is because of the too old version of Gradle that doesn't support newest Java versions. I've decided to update it in occasion.